carts-and-checkout/carts/cart-items/tax-items/create-a-tax-item.md

```diff
@@ -0,0 +1,100 @@
+# Create a Tax Item
+
+{% api-method method="post" host="https://api.moltin.com" path="/v2/carts/:cartID/items/:itemID/taxes" %}
+{% api-method-summary %}
+Create a Tax Item
+{% endapi-method-summary %}
+
+{% api-method-description %}
+
+{% endapi-method-description %}
+
+{% api-method-spec %}
+{% api-method-request %}
+{% api-method-headers %}
+{% api-method-parameter name="Authorization" type="string" required=true %}
+The Bearer token to grant access to the API
+{% endapi-method-parameter %}
+{% endapi-method-headers %}
+
+{% api-method-body-parameters %}
+{% api-method-parameter name="type" type="string" required=true %}
+This represents the type of object being returned
+{% endapi-method-parameter %}
+
+{% api-method-parameter name="name" type="string" required=true %}
+The name of the tax item
+{% endapi-method-parameter %}
+
+{% api-method-parameter name="code" type="string" required=true %}
+A unique tax code for this kind of tax item
+{% endapi-method-parameter %}
+
+{% api-method-parameter name="rate" type="float" required=true %}
+The tax rate percentage represented as a float
+{% endapi-method-parameter %}
+{% endapi-method-body-parameters %}
+{% endapi-method-request %}
+
+{% api-method-response %}
+{% api-method-response-example httpCode=201 %}
+{% api-method-response-example-description %}
+
+{% endapi-method-response-example-description %}
+
+```javascript
+{
+    "data": {
+        "id": "003e2458-3415-4fd2-a10c-ed422bfac4bb",
+        "type": "tax_item",
+        "name": "Tax Name",
+        "jurisdiction" : "UK",
+        "code": "MYTAX01",
+        "rate": 0.2
+    }
+}
+```
+{% endapi-method-response-example %}
+
+{% api-method-response-example httpCode=422 %}
+{% api-method-response-example-description %}
+<<<<<<< HEAD
```

Comment:
Merge conflict 😆

Reply:
Disappointed in myself now, I don't know how this didn't flag in my rebase.

Reply:
Oh... I didn't save my change to the file. Doh.